src/core/server/saved_objects/import/types.ts

```diff
@@ -176,3 +177,72 @@ export interface SavedObjectsResolveImportErrorsOptions {
 }
 
 export type CreatedObject<T> = SavedObject<T> & { destinationId?: string };
+
+/**
+ * A simple informative warning that will be displayed to the user.
+ *
+ * @public
+ */
+export interface SavedObjectsImportSimpleWarning {
+  type: 'simple';
+  /** The translated message to display to the user */
+  message: string;
+}
+
+/**
+ * A warning meant to notify that a specific user action is required to finalize the import
+ * of some type of object.
+ *
+ * @remark The `actionUrl` must be a path relative to the basePath, and not include it.
+ *
+ * @public
+ */
+export interface SavedObjectsImportActionRequiredWarning {
+  type: 'action_required';
+  /** The translated message to display to the user. */
+  message: string;
+  /** The path (without the basePath) that the user should be redirect to to address this warning. */
+  actionUrl: string;
```

Comment:
Should we name this `actionPath` instead of URL to make this more clear that it should be an in-app path and cannot be an external URL?